Makefile

```diff
@@ -484,10 +484,19 @@ test-and-cover: $(RESOURCES_DOT_GO)
 		source envvars.bash; \
 	fi && \
 	for pkg in $$(cat testpackages.txt); do \
-		go test -v -tags="headless" -covermode=count -coverprofile=profile_tmp.cov $$pkg || exit 1; \
+		go test -race -v -tags="headless" -covermode=count -coverprofile=profile_tmp.cov $$pkg || exit 1; \
 		tail -n +2 profile_tmp.cov >> profile.cov; \
 	done
 
+test: $(RESOURCES_DOT_GO)
```

Comment:
New target for testing without coverage (so we get good line numbers on race detection)